modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/GraphQLServerGenerator.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ * Copyright 2018 SmartBear Software
```

Comment:
Please remove line 3 as it's a new file after the fork: https://opensource.stackexchange.com/questions/7300/copyright-notice-in-the-file-header-apache-v2-license

Reply:
was copy'n' paste error - is removed with next commit